docs/markdown/Reference-tables.md

```diff
@@ -67,6 +69,7 @@ set in the cross file.
 | arc                 | 32 bit ARC processor     |
 | arm                 | 32 bit ARM processor     |
 | e2k                 | MCST Elbrus processor    |
+| TMS320C28x          | 32 bit C2000 processor   |
```

Comment:
This seems like a too exact name for a CPU family. Going through the docs would seem to indicate something like `c28x` or something. `TMS320C28x` is more of a value for `cpu` rather than `cpu_family`.

Reply:
yeah I can go with `c28x` on this